README.md

```diff
@@ -43,7 +43,7 @@ Learn all about Liquibase by taking our free online courses at [Liquibase Univer
 
 ## Want to help?
 
-Want to file a bug or improve documentation? Excellent! Read up on our guidelines for [contributing](https://www.liquibase.org/community/index.html)!
+Want to file a bug or improve documentation? Excellent! Read up on our guidelines for [contributing](https://www.liquibase.com/community/contributors)!
```

Comment:
Hey @kevin-atx,

I just saw you also made some of these changes in the `README` file. In this case, I see the link you updated points to `https://contribute.liquibase.com/` instead of `https://www.liquibase.com/community/contributors ` as proposed by @bartipan. Which one should we go with?

Reply:
I'd prefer the contribute site since it's talking about filing a bug or improving docs.